<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="hs_time">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_seconds">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="intValue" stdset="0">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_milliseconds">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>165</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Baud rate</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="rb_1200Baud">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1200</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="rb_19200Baud">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>19200</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="rb_2400Baud">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2400</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="rb_38400Baud">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>38400</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="rb_4800Baud">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4800</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="rb_57600Baud">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>57600</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="rb_9600Baud">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>9600</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="rb_115200Baud">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>115200</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>128</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>COM Port</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QComboBox" name="cb_serialPorts"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Data bits</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QRadioButton" name="rb_5DataBits">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="rb_7DataBits">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="rb_8DataBits">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="rb_6DataBits">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>83</width>
      <height>132</height>
     </rect>
    </property>
    <property name="title">
     <string>Parity</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QRadioButton" name="rb_parityNone">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>none</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="rb_parityOdd">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>odd</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="rb_parityEven">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>even</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="rb_parityMark">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>mark</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="rb_paritySpace">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>space</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Stop bits</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="2" column="0">
      <widget class="QRadioButton" name="rb_2StopBit">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="rb_15StopBit">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1.5</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="rb_1StopBit">
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="tb_serialConsole">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>441</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_connectSerialPort">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Playback speed:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_recent"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>Play</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionFast_forward"/>
    <addaction name="actionRewind"/>
    <addaction name="separator"/>
    <addaction name="actionRecord"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionConfigure_serial_device"/>
    <addaction name="separator"/>
    <addaction name="actionLoop_playback"/>
    <addaction name="separator"/>
    <addaction name="actionFloat_on_top"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionStretch_Compress_time"/>
    <addaction name="actionTrim_empty_data"/>
    <addaction name="actionExtract_segment"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlay"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionOpen_recent">
   <property name="text">
    <string>Open recent</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionFast_forward">
   <property name="text">
    <string>Fast forward</string>
   </property>
  </action>
  <action name="actionRewind">
   <property name="text">
    <string>Rewind</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionConfigure_serial_device">
   <property name="text">
    <string>Configure serial device</string>
   </property>
  </action>
  <action name="actionLoop_playback">
   <property name="text">
    <string>Loop playback</string>
   </property>
  </action>
  <action name="actionFloat_on_top">
   <property name="text">
    <string>Float on top</string>
   </property>
  </action>
  <action name="actionStretch_Compress_time">
   <property name="text">
    <string>Stretch/Compress time</string>
   </property>
  </action>
  <action name="actionTrim_empty_data">
   <property name="text">
    <string>Trim empty data</string>
   </property>
  </action>
  <action name="actionExtract_segment">
   <property name="text">
    <string>Extract segment</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
